parseAndPopulate/modules.py

```diff
@@ -351,9 +352,18 @@ def __resolve_imports(self, git_commit_hash):
         except:
             return
 
-    def add_vendor_information(self, platform_data,
-                               confarmance_type, capability, netconf_version,
-                               integrity_checker, split):
+    def add_vendor_information(self, platform_data, conformance_type, capability,
+                                netconf_version, integrity_checker, split):
```

Comment:
add types to method arguments if you are creating docs for them as well. Also add these every time we create new function. We need to start implementing them slowly everywhere.. It doesn t mean that you have to go through whole file and all files and fix it this way.